Resources/Prototypes/Entities/Markers/spawn_points_jobs.yml

```diff
@@ -1,7 +1,7 @@
 # Base
 - type: entity
   id: SpawnPointJobBase
-  parent: spawn_point
+  parent: SpawnPoint
```

Comment:
You need to update stationstation as well if you're gonna change this.

Reply:
Done